src/libasr/pass/openmp.cpp

```diff
@@ -3251,7 +3250,7 @@ class ParallelRegionVisitor :
                     total_size += 8;
                 }
             }
-            return total_size + STRUCT_VTABLE_SIZE;
+            return total_size + ASRUtils::VTABLE_PTR_SIZE;
```

Comment:
```suggestion
            return total_size;
```
If vptr doesn't exist in openmp structs (C structs) -- This change should work.

Reply:
It worked.